doc/classes/ShapeCast2D.xml

```diff
@@ -75,28 +75,28 @@
 			<return type="bool" />
 			<argument index="0" name="layer_number" type="int" />
 			<description>
-				Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.
+				Returns true if the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.
```

Comment:
This construct is actually used consistently for all mask/layer methods, it would be better to leave it unchanged, or change them all if that's really needed:
```
$ rg "Returns whether or not" -g'*.xml'
doc/classes/NavigationAgent2D.xml
41:                             Returns whether or not the specified layer of the [member navigation_layers] bitmask is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/NavigationAgent3D.xml
41:                             Returns whether or not the specified layer of the [member navigation_layers] bitmask is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/OccluderInstance3D.xml
20:                             Returns whether or not the specified layer of the [member bake_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/VisualInstance3D.xml
39:                             Returns whether or not the specified layer of the [member layers] is enabled, given a [code]layer_number[/code] between 1 and 20.

doc/classes/NavigationRegion2D.xml
21:                             Returns whether or not the specified layer of the [member navigation_layers] bitmask is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/NavigationRegion3D.xml
28:                             Returns whether or not the specified layer of the [member navigation_layers] bitmask is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/RayCast2D.xml
60:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/NavigationMesh.xml
37:                             Returns whether or not the specified layer of the [member geometry_collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/RayCast3D.xml
61:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/SoftDynamicBody3D.xml
31:                             Returns whether or not the specified layer of the [member collision_layer] is enabled, given a [code]layer_number[/code] between 1 and 32.
38:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/ShapeCast2D.xml
78:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/CollisionObject3D.xml
34:                             Returns whether or not the specified layer of the [member collision_layer] is enabled, given a [code]layer_number[/code] between 1 and 32.
41:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/CollisionObject2D.xml
33:                             Returns whether or not the specified layer of the [member collision_layer] is enabled, given a [code]layer_number[/code] between 1 and 32.
40:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.

doc/classes/Camera3D.xml
36:                             Returns whether or not the specified layer of the [member cull_mask] is enabled, given a [code]layer_number[/code] between 1 and 20.

modules/gridmap/doc_classes/GridMap.xml
60:                             Returns whether or not the specified layer of the [member collision_layer] is enabled, given a [code]layer_number[/code] between 1 and 32.
67:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.
80:                             Returns whether or not the specified layer of the [member navigation_layers] bitmask is enabled, given a [code]layer_number[/code] between 1 and 32.

modules/csg/doc_classes/CSGShape3D.xml
18:                             Returns whether or not the specified layer of the [member collision_layer] is enabled, given a [code]layer_number[/code] between 1 and 32.
25:                             Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.
```

We do use `"Returns whether"` a lot for boolean methods, though the "or not" seems redundant indeed.

Reply:
Okay, I'll change it back.